### ISSUE TITLE
PHP 8 Warnings when assignee in TODOLIST has no todos

### ISSUE THREAD

**Issue by arlt** [leibler/dokuwiki-plugin-todo#139]
```
~~TODOLIST ns:test:plugins assigned:arlt~~
```

if nothing is assigned, i get a warning in log:

```
AH01071: Got error 'PHP message: PHP Warning:  count(): Parameter must be an array or an object that implements Countable in /var/www/dokuwiki/lib/plugins/todo/syntax/list.php on line 322'
```
Dokuwiki log:
```
TypeError: count(): Argument #1 ($value) must be of type Countable|array, null given /var/www/dokuwiki/lib/plugins/todo/syntax/list.php(322)
```
Dokuwiki doesn't work anymore.
If i remove the box plugin from the plugin folder, all works fine again.

Dokuwiki Hogfather + PHP 7.4 works
Dokuwiki Igor + PHP 7.4 works
Dokuwiki Igor + PHP 8.1 fails

maybe related to #121

**Comment by runout-at** [comment]
Do you use the plugin version which includes the commit https://github.com/leibler/dokuwiki-plugin-todo/commit/cbf034ab54df05c242f1cfcdedb512b457f3dedb which fixes #121?

This fix is in the Master-Branch. Still not in the Stable-Branch.

I will close this as a duplicate of #121

**Comment by runout-at** [comment]
I just did a merge from master to stable.

The issue was already in stable but some other fixes not. Please upgrade the plugin and check again if this issue is fixed (i have no PHP8 running ).

**Comment by arlt** [comment]
> Do you use the plugin version which includes the commit [cbf034a](https://github.com/leibler/dokuwiki-plugin-todo/commit/cbf034ab54df05c242f1cfcdedb512b457f3dedb) which fixes #121?

Yes, the fix was already present.

**Comment by arlt** [comment]
I commented in #121 - but this issue is closed, too.
